backend/question-service/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:20
```

Comment:
Just curious for all the Dockerfiles that you created, do you use any tool that auto-generates the code? If so, can you let me know which tool did you use? Thank you!

Reply:
nope, i didn't use any tools 😄

Reply:
I see! Thanks for creating all the Dockerfile from scratch, awesome bro!